static/app/utils/makeSafeRefluxStore.ts

```diff
@@ -0,0 +1,50 @@
+export interface SafeRefluxStore extends Reflux.Store {
+  teardown(): void;
```

Comment:
Can we add a docblock comment to this to describe what `teardown` is used for?

Reply:
I'll add it to the implementation, but good call yes!

Reply:
Let’s add it to the member here of the interface actually, so we can get documentation in the language server hover tooltip :)